module/.golangci.yml

```diff
@@ -1,36 +1,28 @@
 linters:
   enable:
-    # - bodyclose
+      - bodyclose
       - deadcode
-    # - depguard
-    # - dogsled
-    # - dupl
-    # - errcheck
-    # - exhaustivestruct
-    # - gochecknoinits
-    # - goconst
-    # - gocritic
-    # - gocyclo
-    # - godox
-    # - gofmt
-    # - goimports
-    # - gosec
```

Comment:
Any plans to add gochecknoinits in the future? I didn't see it commented out below

Reply:
we've been going down the list of lints that where in this file when we started (presumably from the stargate template although I haven't actually checked the history as to when this file originated). We can add that one commented out as a note to enable it in the future.